ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -1154,6 +1154,12 @@ ompi_mtl_ofi_finalize(struct mca_mtl_base_module_t *mtl)
     free(ompi_mtl_ofi.comm_to_context);
     free(ompi_mtl_ofi.ofi_ctxt);
 
+    /* This was strdup()ed at L.714 */
```

Comment:
It looks like this is 1024. but having line numbers is always dicey. I'd say remove the line reference.